.travis.yml

```diff
@@ -19,9 +19,9 @@ branches:
 
 env:
   matrix:
-    - RUN_LINT=true
-    - RUN_FRONTEND_TESTS=true
-    - RUN_BACKEND_TESTS=true REPORT_BACKEND_COVERAGE=false EXCLUDE_LOAD_TESTS=true
+    # - RUN_LINT=true
```

Comment:
Here and elsewhere in this file, please add a note to say that these lines are commented out because these checks are being run on Circle-CI. Include a link to the Circle-CI page for Oppia.

(I mentioned something similar in a [previous comment](https://github.com/oppia/oppia/pull/6234/files#r255209240) which does not seem to have been addressed.)

Reply:
My bad, I missed this. I have added the comments as suggested. Please note that the link will work once we merge this PR, that is, have a config for circle ci and authorize circle ci for the repo.

Reply:
Awesome! Can you give me the steps for the latter (maybe send an email)? Then I'll do it once this PR is merged.